src/pages/home/components/Sidebar/components/SocialAccordion/SocialAccordion.tsx

```diff
@@ -26,6 +26,13 @@ function SocialAccordion(props: SocialAccordionProps): ReactNode {
     if (!enabled) removeAccount(account.id);
   };
 
+  const favoriteAccount = async (
+    isFavorited: boolean,
+    account: StoreAccount
+  ): Promise<void> => {
+    await updateAccount({ ...account, favorite: isFavorited });
+  };
```

Comment:
Essa funçào a mesma coisa, não precisa de async aqui, o componente vai ser atualizado de forma async e tudo bem, não precisamos aguardar nada aqui!

Se o problema é lint vc pode usar 
```suggestion
  const favoriteAccount = (
    isFavorited: boolean,
    account: StoreAccount
  ): void => {
    void updateAccount({ ...account, favorite: isFavorited });
  };
```